src/styles/utils/_loops.scss

```diff
@@ -0,0 +1,26 @@
+@for $day from 1 through 31 {
```

Comment:
I am not sure but instead of loop.scss these all loops can go in respective blocks.

Reply:
Okay, changed.